src/test/java/com/kenai/jffi/VersionTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kenai.jffi;
+
+import org.junit.Test;
+
+import com.kenai.jffi.InvokerTest.HeapInvoker;
+import com.kenai.jffi.InvokerTest.NativeInvoker;
+import com.kenai.jffi.UnitHelper.Address;
+
+import static org.junit.Assert.assertEquals;
+
+public class VersionTest {
+
+    public VersionTest() {
+    }
+
+    @Test public void old_answer() {
+        Invoker invoker = new NativeInvoker();
```

Comment:
Should we use NativeInvoker, HeapInvoker, both, or do this a different way?

Reply:
Whatever works for the test is fine.